style.css

```diff
@@ -1,3 +1,83 @@
 #root {
-  color: red;
+  max-width: 900px;
+  display: flex;
+  justify-content: center;
+  margin: 0 auto;
+  flex-wrap: wrap;
 }
+body{
+  background-color: #F4ECE6;
+}
+
+
+* {
+  box-sizing: border-box;
+}
+
+.episode-title{
+  box-shadow: 3px 3px #1D0029;
+  border-radius: 5px;
+  padding: 5px;
+  background-color: #1D0029;
+  color: whitesmoke;
+ 
+} 
+.episode-title:hover{
+  text-decoration: underline;
+}
+.episode-container{
+ background-color: #1D0029;
+ color: white;
+ width: 31%;
+  text-align: center;
+  margin: 10px;
+  padding: 10px;
+  box-shadow: 3px 3px whitesmoke;
+  border-radius: 10px;
+}
```

Comment:
Because the flex-wrap earlier in your code it would work better on smaller screens if you used max-width instead of width, so that a single episode would appear.